src/components/Toolbar.tsx

```diff
@@ -106,15 +124,15 @@ const Toolbar: FC<ToolbarProps> = ({ customize, onSelect, selected }) => {
   }, shallowEqual)
 
   const onTapCancel = useCallback(() => {
-    setPressingToolbarId(null)
-  }, [])
+    deselectPressingToolbarId()
+  }, [deselectPressingToolbarId])
```

Comment:
`onTapCancel` can be removed entirely.

Reply:
Thank you for pointing this out. Removed: 95acb5ceda43d481ffd62d3fb606544e3a20220f